css/styles.css

```diff
@@ -55,39 +60,307 @@ a {
 	clear: both;
 }
 
+/* Photodiary */
+
+.photo-diary {
+	width: 85%;
+	padding-bottom: 100px;
+	margin: 0 auto;
+	color: rgba(115, 115, 115, .9);
+}
+
+.photo-diary span {
+	display: block;
+	font-size: 0.9rem;
+	font-family: 'Ubuntu', sans-serif;
+	text-transform: uppercase;
+	opacity: .6;
+}
+
+.photo-diary h1 {
+	padding-bottom: 10px;
+	font-family: "Playfair Display", serif;
+	font-weight: normal;
+}
+
+.photo-diary p {
+	display: block;
+	padding-bottom: 40px;
+	font-family: "Playfair Display", serif;
+	font-size: 0.9rem;
+}

```

Comment:
Ustaw sobie większą wysokość linii. Polecam `line-height: 1.5;` bo tekst jest za bardzo ściśnięty w pionie i źle się go czyta. Sprawdziłem w PSD-ku i tam linie są wyższe :) W ogóle to line-height jak i font polecam Ci ustawić dla `body` jako bazę dla wszystkiego :) Czyli:
```css
body {
  font-family: "Playfair Display", serif;
  line-height: 1.5;
}
```
powinno Ci znacznie ułatwić robotę :)

Swoją drogą `.photo-diary p` i `.article p` mają bardzo dużo wspólnych stylów. Wyciągnij je (rozmiar fonta, kolory, itp.) w jedno miejsce - najlepiej ostyluj sobie gdzieś na górze samo `p`.